lab_4_summarization_textrank/main.py

```diff
@@ -146,8 +215,20 @@ def calculate_similarity(sequence: Union[list, tuple], other_sequence: Union[lis
     :param other_sequence: a sequence of items
     :return: similarity score
     """
-    pass
-
+    if not isinstance(sequence, (list, tuple)):
+        raise ValueError
+    if not isinstance(other_sequence, (list, tuple)):
+        raise ValueError
+    if not sequence or not other_sequence:
+        return 0
+    numerator = []
+    denominator = set(tuple(sequence) + tuple(other_sequence))
+    for i in sequence:
+        if i in other_sequence:
+            numerator.append(i)
+    set(numerator)
```

Comment:
`set` does not remove duplicates inplace

Reply:
I'm sorry, I don't understand what do you want me to do...
should i delete duplicates in sequence and other_sequence before adding elements to numerator?

Reply:
it is ok now, thanks!